pkgs/development/python-modules/unify/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchFromGitHub, untokenize }:
+
+buildPythonPackage rec {
+  pname = "unify";
+  version = "0.5";
+
+  # PyPi release is missing tests (see https://github.com/myint/unify/pull/18)
+  src = fetchFromGitHub {
+    owner = "myint";
+    repo = "unify";
+    rev = "v${version}";
+    sha256 = "1l6xxygaigacsxf0g5f7w5gpqha1ava6mcns81kqqy6vw91pyrbi";
+  };
+
+  propagatedBuildInputs = [ untokenize ];
+
+  meta = with lib; {
+    description = "Modifies strings to all use the same quote where possible";
```

```suggestion
    description = "Modifies strings to all use the same quote where possible";
    homepage = "https://github.com/myint/unify";
```

Adding the homepage makes it easier to quickly navigate to the upstream codebase.